### PR TITLE
Some drivers for imx225 and JX-H42

### PR DESCRIPTION
**Pull request by pianist**
imx225: solved gain/explosure problems (may be not for you... :), new resolution 960p.

JS-H42: working driver, tested

### CODE REVIEW

File: general/package/hisilicon-osdrv-hi3516cv100/hisilicon-osdrv-hi3516cv100.mk
Comment (FlyRouter) [inline general/package/hisilicon-osdrv-hi3516cv100/hisilicon-osdrv-hi3516cv100.mk:1]: Okay, let's add

File: general/package/hisilicon-osdrv-hi3516cv100/files/sensor/config/jxh42_i2c_dc_720p.ini
Comment (FlyRouter) [inline general/package/hisilicon-osdrv-hi3516cv100/files/sensor/config/jxh42_i2c_dc_720p.ini:1]: Okay, let's add

File: general/package/hisilicon-osdrv-hi3516cv100/files/sensor/config/imx225_i2c_960p.ini
Comment (FlyRouter) [inline general/package/hisilicon-osdrv-hi3516cv100/files/sensor/config/imx225_i2c_960p.ini:1]: Okay, let's add this file

File: general/package/hisilicon-osdrv-hi3516cv100/files/script/load_hisilicon
Comment (FlyRouter) [inline general/package/hisilicon-osdrv-hi3516cv100/files/script/load_hisilicon:1]: Thanks for the additions and comments

**Review by FlyRouter — COMMENTED** [review]
If the operation of these drivers has been verified, there are backups of the source files, and working testing has also been verified, then we can accept these changes.
Thank you

**Comment by pianist** [comment]
Source files were commited to sersors repo. Eveyone can test driver, I will be happy!

**Review by flyrouter — APPROVED** [review]
Thank you

**Pull request merged**